web/static/js/settings/section-refresh.js

```
@@ -0,0 +1,118 @@
+  // (and callers that want to chain) can await completion; callers in templates
+  // fire-and-forget.
+  function refreshSection(name) {
+    var live = document.querySelector('[data-settings-fragment="' + name + '"]');
+    if (!live) {
+      // Loud failure: a caller asked to refresh a section whose wrapper is not
+      // on the page. Never silently no-op.
+      console.error('swRefreshSettingsSection: no live [data-settings-fragment="' + name + '"] on the page');
+      return Promise.resolve(false);
+    }
+
+    var refreshID = (latestRefreshByName[name] = (latestRefreshByName[name] || 0) + 1);
```

Biome (2.5.1)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)